src/style.css

```
@@ -1 +1,70 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
Don't reset margins and paddings in * selector. It's better to specify the tags you want to reset